src/Core/Enums/ButtonAppearance.cs

```diff
@@ -34,4 +34,3 @@ public enum ButtonAppearance
     /// </summary>
     Transparent,
 }
```

Comment:
Why did you update this file ?

Reply:
I didn't, may be a refactor on save happens on this file